packages/server/package.json

```diff
@@ -21,6 +21,7 @@
     "lint": "set -ex; tsc --build; tslint --project ."
   },
   "dependencies": {
+    "@mdx-js/parcel-plugin-mdx": "^1.0.18",
```

Comment:
Remove this package from `packages/ui`

Reply:
Tried, but it looks like it's needed for the Readme to work.

Reply:
Hmm, then it probably doesn't have to be apart of the server... Can try and publish to see

Reply:
Yeah, it didn't work :(